### DISCUSSION TITLE
Router Question

### DISCUSSION THREAD

**Discussion by flareso** [webfactorymk/flutter-template#44] [Q&A]
Hi why you use custom navigator and managing everything by yourself?
can you explain?
why no use navigaor.of(context),

**Comment by dimitar-zabaznoski** [comment]
Not sure I understand your question. We use Navigator 2.0 for navigation. [Here's a medium article for clarification.](https://medium.com/flutter/learning-flutters-new-navigation-and-routing-system-7c9068155ade)
